packages/wonder-blocks-icon/components/icon.test.js

```diff
@@ -0,0 +1,275 @@
+// @flow
+import * as React from "react";
+import * as Core from "@khanacademy/wonder-blocks-core";
+import icons from "../util/icon-assets.js";
+import {getPathForIcon, viewportPixelsForSize} from "../util/icon-util.js";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+// We mock things out so that we're in control of what really gets rendered.
+// Means we can test that we're using addStyle to generate the component
+// and then verify what that component has going on.
+jest.mock("@khanacademy/wonder-blocks-core", () => {
+    const mockStyledSVGComponent = jest.fn((props: any) => <div>Pretend</div>);
+    return {
+        _mockStyledSVGComponent: mockStyledSVGComponent,
+        addStyle: jest.fn(() => mockStyledSVGComponent),
+    };
+});
+
+// We cannot make the mock outside of the jest.mock call, so we make it inside
+// then attach it to the exports and grab it here.
+const mockStyledSVGComponent: JestMockFn<any, any> = (Core: any)
+    ._mockStyledSVGComponent;
```

Comment:
Where does `JestMockFn` come from?  Does `flow` just know about it?

Reply:
It's in the `jest` flow-typed definition.